### PR TITLE
pprint++

### PR DESCRIPTION
**Pull request by scopatz**

### CODE REVIEW

File: recipes/pprintpp/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
```
Comment (synapticarbors) [inline recipes/pprintpp/meta.yaml]: Perhaps add a test for the command line tool and also an entry point:

https://github.com/wolever/pprintpp/blob/master/setup.py#L26

Comment (scopatz) [inline recipes/pprintpp/meta.yaml]: Ok added!

File: recipes/pprintpp/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - setuptools
```
Comment (jakirkham) [inline recipes/pprintpp/meta.yaml]: We should be able to drop this after making the change above.

File: recipes/pprintpp/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/wolever/pprintpp/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
```
Comment (scopatz) [inline recipes/pprintpp/meta.yaml]: done

File: recipes/pprintpp/meta.yaml (hunk 2 above)
Comment (scopatz) [inline recipes/pprintpp/meta.yaml]: and done!

**Comment by scopatz** [comment]
Ping @conda-forge/core

**Pull request merged**